### DISCUSSION TITLE
what is the limit of projects for pro plan?

### DISCUSSION THREAD

**Discussion by junhanRyu** [supabase/supabase#21275] [Questions] [answered]
I have checked that free plan has 2 limit of projects. but i caanot find the information for pro plan. please let me know

**Comment by GaryAustin1** [comment, marked as answer]
I'm not sure there is a max limit on instances per pro organization.  You basically get your first instance and the set egress and disk size capacity for $25.  For each basic instance you add to the pro organization you get charged for compute time.  The smallest instance is $10 a month.  You can add multiple instances and you just add compute time.  All instances share egress and disk size capacity and billing for overage.

This is all spelled out in pricing  https://supabase.com/pricing

**Reply by MaxHermez** [reply]
Just to confirm, does this mean that we get to keep the 2 free active project AND any project on top of that we pay per instance? Or do we lose access to free projects in that case? (which wouldn't make sense, but I have to make sure)

**Reply by GaryAustin1** [reply]
You get the two free projects but they need to be in their own Free org(s).
